services/axiosInstance.ts

```diff
@@ -7,6 +8,32 @@ export const instance = axios.create({
   headers: { 'Content-Type': 'application/json' },
 });
 
+instance.interceptors.request.use(async (config) => {
+  if (config.url === 'auth/refresh') return config;
+
+  const response =
+    await axios.get<Record<string, string | null>>('/api/cookies');
+  const cookies = response.data;
+
+  try {
+    if (Object.values(cookies).some((cookie) => cookie === null))
+      throw new AxiosError('저장된 유저 정보가 없습니다.', '401');
+
+    const { accessToken } = await postRefresh({
+      refreshToken: cookies.refreshToken!,
+    });
+    await axios.patch('/api/cookies', { accessToken });
+    cookies.accessToken = accessToken;
+  } catch {
+    await axios.delete('/api/cookies');
+    cookies.accessToken = null;
+  }
+
+  if (cookies.accessToken)
+    config.headers['Authorization'] = `Bearer ${cookies.accessToken}`;
```

Comment:
괄호 추가 어떠신가요 ㅋㅋㅋㅋ
아니면 if문없이 header추가하는부분 위에 try블록 밑에 있어도 될것같습니답!
```suggestion
  if (cookies.accessToken) {
    config.headers['Authorization'] = `Bearer ${cookies.accessToken}`;
  }
```

Reply:
아 try에서 오류가 안나면 바로 헤더에 넣어도 문제 없었겠네요🤔 개선안 감사합니다!